src/utils/envConfig.js

```diff
@@ -0,0 +1,49 @@
+export function fetchEnvData() {
+    const envDefined = (typeof process !== "undefined") && process.env;
+    if (envDefined) {
+        let envKeys = Object.keys(process.env);
+        let arrLoaded = envKeys.filter(item => {
+            let regex = RegExp("react_", "gi");
+            return regex.test(item);
```

Comment:
I think this should work, and is a little more obvious/succinct
```suggestion
            return item.startsWith("REACT_");
```

Reply:
thank you, fixed